content/_authors/mauko.quiroga.md

```diff
@@ -1,19 +1,19 @@
 ---
-fullname: Mauko Quiroga
-role: Entrepreneur public
-domaine: Animation
-link: https://www.linkedin.com/in/maukoquiroga/
+fullname: Mauko Quiroga Alvarado
+role: Autre
+domaine: Autre
+link: https://hacerestado.org
 missions:
   - start: 2016-02-15
-    end: 2021-03-04
-    status: admin
-    employer: dinum
+    end: 2023-01-31
```

Comment:
You should probably append a new value instead of editing the existing one :)

Reply:
@maukoquiroga I confirm, you have to append a new mission :)